client/src/pages/AdminLogin/index.jsx

```diff
@@ -14,20 +17,23 @@ import Copyright from '../../components/Copyright';
 import loginAdmin from './apiCalls';
 
 const AdminLogin = () => {
-  const [helperText, setHelperText] = useState('');
-
   const classes = useStyles();
   const dispatch = useDispatch();
 
-  const { SET_LOGIN } = ADMIN_ACTIONS;
-  const { OPEN_SNACKBAR } = SNACKBAR_ACTIONS;
+  const { SET_LOGIN, SET_HELPER_TEXT } = ADMIN_ACTIONS;
+  const { UPDATE_SNACKBAR } = SNACKBAR_ACTIONS;
```

Comment:
again, rather than creating an intermediate value, you can just destructure straight from the import. e.g. `import { SET_LOGIN, SET_HELPER_TEXT } from '../../store/actions/adminLoginActions'`.

Reply:
Having issues getting this working, on compiling I receive
`Attempted import error: 'SET_HELPER_TEXT' is not exported from '../../store/actions/adminLoginActions'.`

Reply:
See my comment on the similar problem above, you may be using default exports when you need named exports

https://exploringjs.com/impatient-js/ch_modules.html#overview-syntax-of-ecmascript-modules

Reply:
Resolved after export restructuring although I had to remove eslint error by comment!